resources/ts/Pages/Integrations/Create.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { FormEvent, ReactNode } from 'react';
+import { useForm } from '@inertiajs/inertia-react';
+import Layout from '../../Shared/Layout';
+import { Heading } from '../../Shared/Heading';
+
+type Props = {
+  integrationTypes: string[];
+  subscriptions: { id: string; name: string }[];
+};
+
+const Index = ({ integrationTypes, subscriptions }: Props) => {
+  const { data, setData, errors, post, processing } = useForm({
+    integrationType: '',
+    subscriptionId: '',
+    name: '',
+    description: '',
+    firstNameOrganisation: '',
+    lastNameOrganisation: '',
+    emailOrganisation: '',
+    firstNamePartner: '',
+    lastNamePartner: '',
+    emailPartner: '',
+  });
+
+  function handleSubmit(e: FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+    post('/integrations');
+  }
+
+  return (
+    <div>
+      <Heading level={2}>Integratie toevoegen</Heading>
+
+      <form onSubmit={handleSubmit}>
+        <h4>Type integratie</h4>
+        <select
+          name="integrationType"
+          id="integrationType"
+          value={data.integrationType}
+          onChange={(e) => setData('integrationType', e.target.value)}
+        >
+          <option value="">Kies...</option>
+          {integrationTypes.map((integrationType) => (
+            <option value={integrationType} key={integrationType}>
+              {integrationType}
+            </option>
+          ))}
+        </select>
+        {errors.integrationType && (
+          <div className="error">{errors.integrationType}</div>
+        )}
+
+        <h4>Plan</h4>
+        <select
+          name="subscriptionId"
+          id="subscriptionId"
+          value={data.subscriptionId}
+          onChange={(e) => setData('subscriptionId', e.target.value)}
+        >
+          <option value="">Kies...</option>
+          {subscriptions.map((subscription) => (
+            <option value={subscription.id} key={subscription.id}>
+              {subscription.name}
+            </option>
+          ))}
+        </select>
+        {errors.subscriptionId && (
+          <div className="error">{errors.subscriptionId}</div>
+        )}
+
+        <h4>Naam integratie</h4>
+        <input
+          type="text"
+          name="name"
+          id="name"
+          value={data.name}
+          onChange={(e) => setData('name', e.target.value)}
+        />
+        {errors.name && <div className="error">{errors.name}</div>}
+
+        <h4>Doel van de integratie</h4>
+        <input
+          type="text"
+          name="description"
+          id="description"
+          value={data.description}
+          onChange={(e) => setData('description', e.target.value)}
+        />
+        {errors.description && (
+          <div className="error">{errors.description}</div>
+        )}
+
+        <h4>Contact organisatie</h4>
```

Comment:
Are we going to add something to handle translations?
I think it's useful to centralize all copy in 1 file.

Reply:
Indeed I think we'll have to support multiple languages. Even if it's only dutch I'll add it to centralize the copy 👍🏻